src/miniapp.ts

```diff
@@ -0,0 +1,5 @@
+export declare namespace MiniApp {
+  export function getUniqueId(): string;
```

Comment:
Also, right now this is just an interface definition. Shouldn't this be a function something like you had before? `(window as any).MiniApp.getUniqueId()`

Reply:
Both android and ios share the same interface name-calling so we don't need it anymore. Previously, the ios calling is different `(window as any).webkit.messageHandlers.MiniApp`